man/ebuild.5

```diff
@@ -807,6 +807,10 @@ is installed.
 .I test_network
 The package manager may run tests that require an internet connection, even if
 the ebuild has
```

Comment:
The end of this sentence was stolen by test_privileged below.

Reply:
Fixed, sorry.